PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetExtractor.h

```diff
@@ -153,6 +154,7 @@ class AliAnalysisTaskJetExtractor : public AliAnalysisTaskEmcalJet {
   TRandom3*                   fRandomGenerator;                         //!<! Random number generator, used for event + jet efficiency
   TRandom3*                   fRandomGeneratorCones;                    //!<! Random number generator, used for random cones
   AliHFJetsTaggingVertex*     fVtxTagger;                               //!<! class for sec. vertexing
+  AliEventCuts                fEventCuts;                               //!<! Event selection  
```

Comment:
AliEventCuts is already implemented in the base class `AliAnalysisTaskEmcal` under `fAliEventCuts`. Just configure that object. See: https://github.com/alisw/AliPhysics/blob/80bac393b634e1e5f344a0c0e71cdb2eb379b2f0/PWG/EMCAL/EMCALbase/AliAnalysisTaskEmcal.h#L1409